teradatalabs/cdh5-hive-kerberized/Dockerfile

```diff
@@ -75,9 +75,11 @@ ADD files/conf/hiveserver2-site.xml /etc/hive/conf/hiveserver2-site.xml
 # CREATE PRESTO PRINCIPAL AND KEYTAB
 RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-server/presto-master@LABS.TERADATA.COM"
 RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-client/presto-master@LABS.TERADATA.COM"
+RUN /usr/sbin/kadmin.local -q "addprinc -randkey hive/presto-master@LABS.TERADATA.COM"
 RUN mkdir -p /etc/presto/conf
 RUN /usr/sbin/kadmin.local -q "xst -norandkey -k /etc/presto/conf/presto-server.keytab presto-server/presto-master"
 RUN /usr/sbin/kadmin.local -q "xst -norandkey -k /etc/presto/conf/presto-client.keytab presto-client/presto-master"
+RUN /usr/sbin/kadmin.local -q "xst -norandkey -k /etc/presto/conf/presto-hive.keytab hive/presto-master"
```

Comment:
The filename `presto-hive.keytab` is a little confusing. Is this name auto-generated or did you specify it? If it is the latter, does it make sense to rename this to say `hive-presto-master.keytab` or something that makes it more clear?